datafusion/sqllogictest/test_files/string_view.slt

```diff
@@ -913,7 +913,7 @@ EXPLAIN SELECT
 FROM test;
 ----
 logical_plan
-01)Projection: find_in_set(CAST(test.column1_utf8view AS Utf8), Utf8("a,b,c,d")) AS c
```

Comment:
I think we should probably add a test that actually invokes `find_in_set` with a `StringVieAwArray` argument.

However, I think @2010YOUY01  has some ideas on a more general testing framework, so maybe this is good enough for now. Let me know what you think

https://github.com/apache/datafusion/issues/11790#issuecomment-2284100621

Reply:
I will add some sqllogictests to make sure it is covered.

Reply:
Done! https://github.com/apache/datafusion/pull/11970/commits/5d5a4e6ab09bcef4619eff2557c3ee71d37da2bd